### ISSUE TITLE
Plex broken

### ISSUE THREAD

**Issue by florisi** [MikeOne/nl.kikkert.plex#6]
After firmware update to 0.10.0 Plex is broken.

**Comment by michielkikkert** [comment]
Yes - I'm aware unfortunately. Athom made a change to the API, however, I'm not sure how to cope with this change (or what the change was in the first place). At the moment, I'm waiting for response from Athom to clarify what I should do to fix this.

**Comment by florisi** [comment]
OK, thanks for the reply.
Good luck with fixing the issue ;)

**Comment by michielkikkert** [comment]
Once I hear from Athom, I will update this thread: https://forum.athom.com/discussion/366/homey-plex-application-main-discussion-thread#latest
